.github/workflows/python-test.yml

```diff
@@ -5,14 +5,15 @@ on:
     branches: [ master ]
   pull_request:
     branches: [ master ]
-
+  schedule:
+    - cron: '30 5 * * *'
 jobs:
   build:
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-22.04
     strategy:
       max-parallel: 4
       matrix:
-        python: [3.5, 3.6, 3.7, 3.8, 3.9]
+        python: [3.8, 3.9]
```

Comment:
We are claiming support for python 3.5-3.7 as well, but we are not testing it. 
Out of those, 3.7 is still supported version, so we shouldn't drop that one.
I would add another job that runs the older python version(s)...